<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<!-- Load params for communication client -->
	<rosparam command="load" file="$(find kuka_rsi3_server)/config/CommunicationClientSetting.yaml" />

	<node name="communication_client" pkg="kuka_rsi3_server" type="communication_test" output="screen">
	</node>

</launch>